p2p/peer.go

```diff
@@ -331,7 +331,8 @@ func (p *peer) SendEnvelope(e Envelope) bool {
 	}
 	res := p.Send(e.ChannelID, msgBytes)
 	if res {
-		p.metrics.MessageSendBytesTotal.With("message_type", metricLabelValue).Add(float64(len(msgBytes)))
+		p.metrics.MessageSendBytesTotal.With("message_type",
+			metricLabelValue, "chID", fmt.Sprintf("%#x", e.ChannelID)).Add(float64(len(msgBytes)))
```

Comment:
[question] Looks like `"%#x"` will print the ChannelID in hexadecimal. Is that the common format used to display ChannelIDs? 

[question] Is there a map from Channel ID => reactor somewhere?

Reply:
> [question] Looks like "%#x" will print the ChannelID in hexadecimal. Is that the common format used to display ChannelIDs?

Yes, it is the format used in other Prometheus metrics labels, please see https://github.com/celestiaorg/celestia-core/blob/d344f2712d38e073b06fdf97d4eecf3634fc027e/p2p/peer.go#L528

Reply:
> [question] Is there a map from Channel ID => reactor somewhere? 

I am not aware of any specs for this, however, every reactor implements the  the `reactor` interface due to which it also supports the  [`GetChannels`](https://github.com/celestiaorg/celestia-core/blob/f7b5c1f8eb71d7dbf8495a3fc69a4a01a2297b53/p2p/base_reactor.go#L23) method which returns the list of supported channel ID for that reactor. Below is an example for the mempool v0 reactor: https://github.com/celestiaorg/celestia-core/blob/3f3b7cc57f5cfc5e846ce781a9a407920e54fb72/mempool/v0/reactor.go#L126

Reply:
Sounds good, thanks! 

I assume a map from channel ID => reactor name will be useful after this merges but I don't think we want to maintain one so I think this thread can be resolved.